### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by smartm0use** [vivirenremoto/doomcaptcha#13]
Hello, maybe I'm missing the point. Can you tell me how to use it?

This is the code for the example:

```
<form action="success.html">
    <p>
        Your Email<br>
        <input name="email">
    </p>
 
    <script src="https://vivirenremoto.github.io/doomcaptcha/script.js?version=16" countdown="on" label="Captcha" enemies="4"></script>

    <p>
        <button>Unsubscribe</button>
    </p>
</form>
```

I mean, for example, how to disable submit button of the form and then enable it only if the captcha is accomplished?

Thank you.

**Comment by Wuelle** [comment]
You can do something like this
https://github.com/vivirenremoto/doomcaptcha/blob/0e426e4acff4f2787978985162fee13ad021f4f6/script.js#L22-L27

So to disable/enable the button you would define the button like this:
```
<button id="submit" disabled>Submit</button>
```
and in the EventListener above you would just add the line
```
document.getElementById('submit').disabled = false;
```
to enable the button

**Comment by vivirenremoto** [comment]
hi @smartm0use ,

in the demo page you can click the submit button, but it doesnt send the form until you complete the captcha, I thought it would be a good idea to highlight the captcha to red to help the user understand that he must complete the captcha first.

the solution by @Wuelle is a good one, but you shoud upload the doom captcha in your server.

i'm not sure if this helps you, but thanks for your comment

**Comment by smartm0use** [comment]
Thank you for your replies.
Instead, what about integrating it in Wordpress (Contact Form 7)? Any way to have it?

**Comment by vivirenremoto** [comment]
@smartm0use i think Its only copy & paste the Doom captcha JavaScript line (the blue line) before the [submit] tag of WP Contact Form and save it....

**Comment by smartm0use** [comment]
Yes, pasting the script before [submit] tag does the trick. Anyway I also tried to add some custom javascript to enable/disable button or to prevent form to be submitted, but custom logic seems not to work within Contact Form 7...
